src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCounter {
+    private final String colour;

```

Comment:
```suggestion
    private final String color;
```
Check  it everywhere